### PR TITLE
Add a simple example of a test written in ansible

### PR DESCRIPTION
**Pull request by psss**
Just to demonstrate how the `shell` framework can be used for this.

### CODE REVIEW

File: examples/ansible/test.fmf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+summary: Run a simple test written in ansible
+test: ansible-playbook -c local test.yml || exit 1
+require: ansible
```
Comment (lukaszachy) [inline examples/ansible/test.fmf:3]: Could you add an explicit `framework: shell` ?

Comment (psss) [reply examples/ansible/test.fmf:4]: Sure, added.

**Comment by psss** [comment]
Rebased to fix the `click` regression which caused tests to fail and squashed. @lukaszachy, any other comments?

**Review by lukaszachy — APPROVED** [review]

**Pull request merged**